pkg/apis/core/types.go

```diff
@@ -2081,6 +2081,9 @@ const (
 	// DNSDefault indicates that the pod should use the default (as
 	// determined by kubelet) DNS settings.
 	DNSDefault DNSPolicy = "Default"
+
+	// DNSCustom indicates that the pod should use an empty DNS setting.
```

Comment:
empty DNS settings. DNS parameters such as nameservers and search paths should be customized via DNSParams

Reply:
Done.